Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -695,11 +690,12 @@ private static string GetBodyFromResponse(HttpWebResponse response)
             return responseBody;
         }
 
-        private HttpWebRequest CreateRequestForPath(string path)
+        private HttpWebRequest CreateRequestForPath(string path, object[] queryObjects = null)
```

Comment:
This function now also accepts an optional array of extra query objects that will be converted to a query string.